### PR TITLE
Feature/pre merged components

### PR DESCRIPTION
**Pull request by kumardennis**

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
          "stylelint-scss"
      ],
      "ignoreFiles": [
-         "src/Assets/**/*"
+         "src/Assets/**/*",
```
Comment (edanchenkov) [inline .stylelintrc.json]: I guess there is no src/Assets folder at all :)

**Review by edanchenkov — CHANGES_REQUESTED** [review]
Why to have both test.png and test.jpeg?

File: stories/Breadcrumb.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+storiesOf('Breadcrumbs', module)
+    .addParameters({
+        backgrounds: [
+            { name: 'light', value: '#fffef0' },
+            { name: 'dark', value: '#1e3b47' },
+            { name: 'purple', value: '#7955ff', default: true }
+        ]
+    })
+    .add('default', () => <Breadcrumbs />);
```
Comment (edanchenkov) [inline stories/Breadcrumb.js]: Of course it would be good to see this component with some items props provided

Comment (kumardennis) [reply stories/Breadcrumb.js]: What props do you want to see?

File: src/Components/HexImage/styles.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
+.hexagon {
+    position: relative;
+    width: 300px;
+    height: 173.21px;
+    margin: 86.6px 0;
+    background-position: center;
+    background-size: auto 346.4102px;
+}
+
+.hexTop,
```
Comment (victorv-sbp) [inline src/Components/HexImage/styles.scss:10]: You should use hyphen instead of camel case for css class names.

**Review by victorv-sbp — APPROVED** [review]

File: src/Components/HexImage/styles.scss (hunk 3 above)
Comment (edanchenkov) [reply src/Components/HexImage/styles.scss:10]: @kumardennis make this as a separate PR to master, this branch was merged.

**Pull request merged**